libs/k8s/custom/core/mapContainers.libsonnet

```
@@ -13,36 +13,40 @@ local patch = {
     |||,
     [d.arg('f', d.T.func)]
   ),
-  mapContainers(f):: {
+  mapContainers(f, includeInitContainers=false):: {
     local podContainers = super.spec.template.spec.containers,
+    local podInitContainers = super.spec.template.spec.initContainers,
     spec+: {
       template+: {
         spec+: {
           containers: std.map(f, podContainers),
+              initContainers: if includeInitContainers then std.map(f, podInitContainers) else podInitContainers,
```

Only modify initContainers if the bool is true. Otherwise, this crashes when `initContainers` is not set and `mapContainer` is called

```suggestion
              [if includeInitContainers then 'initContainers']: std.map(f, podInitContainers),
```

---

Should have fixed this